examples/discrete_hpbandster.py

```diff
@@ -1,29 +0,0 @@
-from tune_sklearn import TuneSearchCV
-from sklearn import datasets
-from sklearn.model_selection import train_test_split
-from sklearn.ensemble import RandomForestClassifier
-import ConfigSpace as CS
-
-digits = datasets.load_digits()
-X = digits.data
-y = digits.target
-
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
-
-space = {
-    "n_estimators": CS.UniformIntegerHyperparameter("n_estimators", 100, 200),
-    "min_weight_fraction_leaf": (0.0, 0.5),
-    "min_samples_leaf": CS.UniformIntegerHyperparameter(
-        "min_samples_leaf", 1, 5)
-}
-
-tune_search = TuneSearchCV(
-    RandomForestClassifier(),
-    space,
-    search_optimization="bohb",
-    n_iter=3,
-    max_iters=10)
```

Comment:
@Yard1 I ended up removing this example because:
1. BOHB requires a form of iterative updating but
2. random forests don't have an iterative update ability (from what I can read - please let me know if this is wrong!)

Happy to add this back, but would want to understand how to make this work practically.

Reply:
I'm fine with it being removed, thanks for working on this.